.github/workflows/ci.yml

```diff
@@ -85,14 +90,40 @@ jobs:
         name: celestia-${{matrix.platform}}
         path: ${{github.workspace}}/build/src/*/*.7z
 
-  build-ubuntu-old:
+  build-posix:
+    name: "${{matrix.config.os}} with ${{matrix.qtversion}}"
     strategy:
       fail-fast: false
       matrix:
-        platform: [ ubuntu-18.04 ]
-    runs-on: ${{matrix.platform}}
+        config:
+        - os: ubuntu-18.04
+          avif: OFF     # Not supported on Ubuntu18.04
+          gles: ON
+          glut: ON
+          gtk3: ON
+          minaudio: OFF # Requires newer g++
+        - os: ubuntu-22.04
+          avif: ON
+          gles: ON
+          glut: ON
+          gtk3: ON
+          minaudio: ON
+        - os: macos-latest
+          avif: ON
+          gles: OFF     # TODO: Figure out how to enable
```

Comment:
only the iOS-on-Mac apps can use gles. Mac apps cannot access GLES APIs, so no need for this TODO